src/puptoo/upload.py

```diff
@@ -1,9 +1,10 @@
 import io
 import json
+import logging
 from minio import Minio
 from .utils import config, puptoo_logging
 
-logger = puptoo_logging.initialize_logging()
+logger = logging.getLogger(config.APP_NAME)
```

Comment:
Inside method `initialize_logging`, it handles cloudwatch related config that allows to app to publish logs to cloudwatch. Is there any other place where it is already giving call to this method on service start?

Reply:
By mistake, I initialized the log here again which has already been done in [app.py](https://github.com/RedHatInsights/insights-puptoo/blob/bd41ff10f0c56183fedcaaa594396d6fc6965f6c/src/puptoo/app.py#L26) file.

Reply:
> Inside method `initialize_logging`, it handles cloudwatch related config that allows to app to publish logs to cloudwatch. Is there any other place where it is already giving call to this method on service start?

It handles clowder+cloudwatch related logs.